geom/alg_convex_hull.go

```diff
@@ -48,108 +48,94 @@ func convexHull(g Geometry) Geometry {
 		return g
 	}
 	pts := g.convexHullPointSet()
-	hull := grahamScan(pts)
-	switch len(hull) {
+	// TODO: length may not be a good signal to indicate
+	// how to convert XY to Geometry. Need to revisit
+	switch len(pts) {
 	case 0:
 		return NewGeometryCollection(nil)
 	case 1:
-		return NewPointXY(hull[0])
+		return NewPointXY(pts[0])
```

Comment:
The `isSame` check below seems as though it can already handle the cases where there are 1 or 2 points. So I don't think this switch is necessary (you would only have to check the 0 points case).